lnrpc/walletrpc/walletkit_server.go

```diff
@@ -178,6 +180,19 @@ var (
 			Entity: "onchain",
 			Action: "write",
 		}},
+		"/walletrpc.WalletKit/SignCoordinatorStreams": {{
+			Entity: "onchain",
+			Action: "write",
+		}, {
+			Entity: "message",
+			Action: "write",
+		}, {
+			Entity: "signer",
+			Action: "generate",
+		}, {
+			Entity: "address",
+			Action: "read",
+		}},
```

Comment:
Wasn't really sure what kind of perms we'd want to require in the macaroon for the remote signer when opening the stream with the watch-only node. Therefore mimicked the perms of the inbound remote signer required permissions. Though as this is the other way around i.e. the remote signer is connecting to the watch-only node, that maybe doesn't makes sense as it's not strictly required.
Perhaps we'd like to add some new kind of entity for this specific case?

Reply:
I think adding a special permission just for the remote signer here probably makes sense.

Reply:
Ok cool, I added a new entity called `remotesigner`, with the action `generate` (I wasn't sure which action made most sense here, so feel free to suggest using read/write if you thin that makes more sense).